src/components/DetailsView/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useContext } from 'react';
+import ModiteContext from '../../state/modite';
+import Modite from '../../models/Modite';
+import classNames from 'classnames/bind';
+import s from './styles.module.css';
+import ModiteProfileResp from '../../models/ModiteProfileResp';
+
+// TODO: type correctly
+function Details({ className = '' } : any) {
+  const [activeModite, setActiveModite]: [Modite, React.Dispatch<any>] = useContext(ModiteContext);
+  const { profile = {} }: any = activeModite || {};
+  let { fields } = profile;
+
+  const fetchProfile = async () => {
+    if (!activeModite) return;
+
+    const moditeProfile: ModiteProfileResp = await fetch(
+      `https://modus.app/modite/${activeModite.id}`,
```

Comment:
Do we ever have these endpoints used locally? If so, we could create a config/env var to change the root url instead of hardcoding modus.app everywhere.

Reply:
I think it's all remote-fed.  Doesn't mean we couldn't sub in an env var, but not it's needed for this project.